src/renderers/webvr/WebVRManager.js

```diff
@@ -441,10 +471,13 @@ function WebVRManager ( renderer ) {
 
 	};
 
-	this.addEventListener = function () { /* Stub */ }
-
 }
 
-Object.assign( WebVRManager.prototype, EventDispatcher.prototype );
+Object.assign( WebVRManager.prototype, {
```

Comment:
Why is this necessary. Should not work with the prototype as before?
Also no need of brackets. Can do I think:

```
addEventListener: EventDispatcher.prototype.addEventListener
```

Reply:
I answered that in https://github.com/aframevr/aframe/pull/5102#discussion_r956064420 EventDispatcher is now a class, and methods of a class are not enumerable.
An alternative is to convert WebVRManager to a class and extends EventDispatcher, but I didn't do that otherwise the diff would have been the whole file.

Reply:
I removed the brackets.